cea/demand/thermal_loads.py

```diff
@@ -1148,12 +1148,12 @@ def get_temperatures(locator, prop_HVAC):
     :type prop_HVAC: Gdf
 
     Sample data (first 5 rows):
-                 Name type_cs type_ctrl type_dhw type_hs
-    0     B154862      T0        T1       T1      T1
-    1     B153604      T0        T1       T1      T1
-    2     B153831      T0        T1       T1      T1
-    3  B302022960      T0        T0       T0      T0
-    4  B302034063      T0        T0       T0      T0
+                 Name type_cs type_ctrl type_dhw type_hs MECH_VENT WIN_VENT HEAT_REC NIGHT_FLSH ECONOMIZER
```

Comment:
in order to keep the number of variables to a minimum i would suggest to add these configurations, and combinations of them, to a a new tipe inside type_cs, type_hs and type_ctrl. This would take the form of T4. T5, T6, T7 etc.. inside the file emission_systems.xls.  In this way, the archetypes do not have to be modified (leaving gaps behind of statistical information). 

We will have to add this types to the documentation and to the excel files